packages/react-aria-components/src/ColorSwatch.tsx

```diff
@@ -33,7 +33,7 @@ export const ColorSwatch = forwardRef(function ColorSwatch(props: ColorSwatchPro
   
   return (
     <div
-      {...mergeProps(DOMProps, renderProps, colorSwatchProps)}
+      {...mergeProps(DOMProps, colorSwatchProps, renderProps)}
```

Comment:
It's opposite order in other components, ie ProgressBar
Are these special cases? or should they all be in this order?

Reply:
it's because `useColorSwatch` returns `style`, and that gets passed into `useRenderProps` as `defaultStyle`, so the user can override it.

Reply:
I checked all the hooks that return styles. I think flipping the order by default would not be enough, because the defaultStyle would have to be passed into useRenderProps anyway.